docs/low-code-sql/development/gems/datasources.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Data Sources
+id: datasources-sql
+description: access your data
+sidebar_position: 1
+tags:
+  - SQL
+  - sources
+  - seeds
+---
+
+Constitutes the set of Gems that help with loading data.
+
+## Source
```

Comment:
Does the Env tab list tables in the warehouse?
Here I have a fabric with the database and schema defined (top image), but my project ENV tab (lower image) can view much more than the scope of the fabric. I don't have any connections defined in the fabric. 
<img width="1231" alt="image" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/372bf5f2-62a6-4592-a7fb-2e2e96b4fb53">

<img width="447" alt="image" src="https://github.com/SimpleDataLabsInc/prophecy-docs/assets/20563764/0a60ef66-5245-4a8b-8424-e2938452bc0b">

Reply:
the fabric is the default write schema

Reply:
so we need to document that the model will write to database.schema as defined in the fabric. 

Can the user overwrite this? Pass as a config?

Reply:
done